addons/repair/models/repair.py

```diff
@@ -115,6 +115,12 @@ def _default_stock_location(self):
     amount_tax = fields.Float('Taxes', compute='_amount_tax', store=True)
     amount_total = fields.Float('Total', compute='_amount_total', store=True)
     tracking = fields.Selection('Product Tracking', related="product_id.tracking", readonly=False)
+    invoice_count = fields.Integer(string='Invoice Count', compute='_get_invoiced')
+
+    @api.depends('invoice_id')
+    def _get_invoiced(self):
+        for repair in self:
+            repair.invoice_count = len(repair.invoice_id)
```

Comment:
`invoice_id` is m2o so Do we need `invoice_count` in this case ?
(or Did I missed something ?)